preact/example-customer-account--loyalty-app--preact/extensions/ca-loyalty-profile-block/src/ProfileBlockExtension.jsx

```diff
@@ -0,0 +1,44 @@
+import '@shopify/ui-extensions/preact';
+import {render} from 'preact';
+
+export default async () => {
+  render(<ProfileBlockExtension />, document.body);
+};
+
+// [START profile.build-ui]
+function ProfileBlockExtension() {
+  const i18n = shopify.i18n;
+
+  return (
+    <s-section heading="Rewards">
+      <s-stack direction="block" gap="base" paddingBlockStart="base">
+        <s-stack direction="inline" gap="large">
```

Comment:
Top is this extension, bottom is the original. I think we should replace this `s-stack` with a `s-grid`. It was originally a `InlineLayout` and those [map](https://shopify.dev/docs/api/checkout-ui-extensions/2025-10-rc/upgrading-to-2025-10#mapping-legacy-components-to-polaris-web-components) to grid

<img width="1161" height="447" alt="Screenshot 2025-09-10 at 10 15 30 AM" src="https://github.com/user-attachments/assets/ce558ac2-9987-4e49-8a56-6214b42b2a4d" />